jest.config.ts

```diff
@@ -11,6 +11,7 @@ const config: Config.InitialOptions = {
   },
   testRegex: '/__tests__/.*.test.ts$',
   verbose: true,
+  snapshotFormat: { escapeString: false, printBasicPrototype: false },
```

Comment:
This becomes the default in jest v29, but I have an issue to open in ts-jest that's blocking that update.